$NetBSD: patch-eeschema_CMakeLists.txt,v 1.6 2022/01/02 19:10:03 tnn Exp $

Don't use ${CMAKE_MODULE_PATH} directly as it will be changed
and contains multiple directories

Force proper LDFLAGS for MODULE, as cmake doesn't set rpath for these

--- eeschema/CMakeLists.txt.orig	2021-12-22 13:32:19.000000000 +0000
+++ eeschema/CMakeLists.txt
@@ -313,8 +313,8 @@ add_custom_command(
     COMMAND ${CMAKE_COMMAND}
         -DinputFile=${CMAKE_CURRENT_SOURCE_DIR}/dialogs/dialog_bom_help.md
         -DoutputFile=${CMAKE_CURRENT_SOURCE_DIR}/dialogs/dialog_bom_help_md.h
-        -P ${CMAKE_MODULE_PATH}/BuildSteps/markdown2C.cmake
-    DEPENDS ${CMAKE_MODULE_PATH}/BuildSteps/markdown2C.cmake ${CMAKE_CURRENT_SOURCE_DIR}/dialogs/dialog_bom_help.md
+        -P ${PROJECT_SOURCE_DIR}/CMakeModules/BuildSteps/markdown2C.cmake
+    DEPENDS ${PROJECT_SOURCE_DIR}/CMakeModules/BuildSteps/markdown2C.cmake ${CMAKE_CURRENT_SOURCE_DIR}/dialogs/dialog_bom_help.md
     COMMENT "creating ${CMAKE_CURRENT_SOURCE_DIR}/dialogs/dialog_bom_help_md.h
        from ${CMAKE_CURRENT_SOURCE_DIR}/dialogs/dialog_bom_help.md"
     )
@@ -330,8 +330,8 @@ add_custom_command(
     COMMAND ${CMAKE_COMMAND}
         -DinputFile=${CMAKE_CURRENT_SOURCE_DIR}/sch_text_help.md
         -DoutputFile=${CMAKE_CURRENT_SOURCE_DIR}/sch_text_help_md.h
-        -P ${CMAKE_MODULE_PATH}/BuildSteps/markdown2C.cmake
-    DEPENDS ${CMAKE_MODULE_PATH}/BuildSteps/markdown2C.cmake ${CMAKE_CURRENT_SOURCE_DIR}/sch_text_help.md
+        -P ${PROJECT_SOURCE_DIR}/CMakeModules/BuildSteps/markdown2C.cmake
+    DEPENDS ${PROJECT_SOURCE_DIR}/CMakeModules/BuildSteps/markdown2C.cmake ${CMAKE_CURRENT_SOURCE_DIR}/sch_text_help.md
     COMMENT "creating ${CMAKE_CURRENT_SOURCE_DIR}/sch_text_help_md.h
        from ${CMAKE_CURRENT_SOURCE_DIR}/sch_text_help.md"
     )
@@ -439,6 +439,9 @@ if( MAKE_LINK_MAPS )
         )
 endif()
 
+set_property(TARGET eeschema_kiface APPEND_STRING
+	PROPERTY LINK_FLAGS " $ENV{LDFLAGS}")
+
 # these 2 binaries are a matched set, keep them together:
 if( APPLE )
     set_target_properties( eeschema PROPERTIES
